x-pack/plugins/apm/common/rules/default_action_message.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const errorCountMessage = i18n.translate(
+  'xpack.apm.alertTypes.errorCount.defaultActionMessage',
+  {
+    defaultMessage: `\\{\\{alertName\\}\\} alert is firing because of the following conditions:
+
+- Service name: \\{\\{context.serviceName\\}\\}
+- Environment: \\{\\{context.environment\\}\\}
+- Threshold: \\{\\{context.threshold\\}\\}
+- Triggered value: \\{\\{context.triggerValue\\}\\} errors over the last \\{\\{context.interval\\}\\}`,
+  }
+);
+
+export const transactionDurationMessage = i18n.translate(
+  'xpack.apm.alertTypes.transactionDuration.defaultActionMessage',
+  {
+    defaultMessage: `\\{\\{alertName\\}\\} alert is firing because of the following conditions:
+
+- Service name: \\{\\{context.serviceName\\}\\}
```

Comment:
Can you test if the escaping is really necessary or if we can do this:

```suggestion
- Service name: {{context.serviceName}}
```

Reply:
We need to scape it otherwise `i18n` breaks:

```
iginal Error: Error: [I18n] Error formatting the default message for: "xpack.apm.alertTypes.errorCount.defaultActionMessage".
      SyntaxError: Expected "0", [1-9] or [^ \t\n\r,.+={}#] but "{" found.
          at Object.translate (i18n.ts:203:11)
```